src/components/Search/CompareWithBase.tsx

```diff
@@ -148,13 +148,25 @@ function CompareWithBase({
   };
 
   const handleRemoveRevisionBase = (item: Changeset) => {
```

Comment:
Wondering if this logic can be simplified for removing the base revision, since we only allow one.

Reply:
yeah, I believe we can have a behavior that's more like radio buttons: instead of warning we could just unselect the old one and select the new one...